runtime/testy/stateManager.py

```diff
@@ -32,12 +32,52 @@ def addPipe(self, processName, pipe):
     self.processMapping[processName] = pipe
     pipe.send(RUNTIME_CONFIG.PIPE_READY.value)
 
-  def getValue(self, key):
-    self.processMapping[PROCESS_NAMES.STUDENT_CODE].send(self.state[key])
+  def getValue(self, keys):
```

Comment:
I think we should put the `*args` logic in these functions inside statemanger. 
That way, the code in the `start()` function will be very clean. Since there are always going to be 2 element lists, we can just do 
cmd_type, args = received
if cmd_type == thing:
  cmd_function(args)